plugins/boltz/swap.go

```diff
@@ -4,10 +4,94 @@ import (
 	"context"
 	"encoding/hex"
 	"fmt"
+	"math"
+	"time"
 
 	"github.com/BoltzExchange/boltz-lnd/boltz"
 )
 
+func (b *Plugin) GetStatus(boltzID string, reverse bool) {
+
+	b.BoltzAPI.SwapStatus(boltzID)
+
+	/*
+			transaction.confirmed: that transaction was included in a block
+		invoice.set: when the invoice of the Submarine Swap was set
+		once the said transaction is included in a block (or found in the mempool in case of 0-confirmation) Boltz will try to pay the invoice provided by the user in order to claim the onchain coins
+		invoice.paid: if paying the invoice was successful
+		invoice.failedToPay: if paying the invoice failed. In which case the locked up onchain coins should be refunded
+		transaction.claimed
+	*/
+
+}
+
+// CalcFundsToReceive calculates how much funds you would receive
+func (b *Plugin) CalcFundsToReceive(ctx context.Context, reverse bool, sats uint64) (uint64, int64, error) {
+	if b.LnAPI == nil {
```

Comment:
Can we remove this part. It is not "CalcFudnsFN" responsibility to initialize lnapi. We should expect it here to be set & ready. otherwise leave it to crash

Reply:
Yeah CalcFundsToReceive will be refactored.